drivers/can/stm32_can.c

```diff
@@ -43,18 +43,47 @@ static inline void can_stm32_get_msg_fifo(CAN_FIFOMailBox_TypeDef *mbox,
 	msg->data_32[1] = mbox->RDHR;
 }
 
-static void can_stm32_isr(void *arg)
+static inline
+void can_stm32_rx_isr_handler(CAN_TypeDef *can, struct can_stm32_data *data)
 {
-	struct device *dev;
-	struct can_stm32_data *data;
-	const struct can_stm32_config *cfg;
-	CAN_TypeDef *can;
-	u32_t bus_off;
+	CAN_FIFOMailBox_TypeDef *mbox;
+	int filter_match_index;
+	struct can_msg msg;
 
-	dev = (struct device *)arg;
-	data = DEV_DATA(dev);
-	cfg = DEV_CFG(dev);
-	can = cfg->can;
+	while (can->RF0R & CAN_RF0R_FMP0_Msk) {
```

Comment:
We usually don't use *_Msk defines.
(One point being the "w/o _Msk define" is commented with bit signification, while the _Msk one is commented with value)
It seems this was present in initial PR, but I did not catch it.
Could you fix this ?

Reply:
I will fix this in my next commit which is a rework of the PR before.
In this PR the bit timing is moved to device tree so that we could support controllers with more than one CAN. Additionally some checks for prescaler and issue #7933 is fixed there.